### PR TITLE
New tests

### PR DESCRIPTION
**Pull request by Oleksandr2070452**

### CODE REVIEW

File: src/test/java/pages/BasePage.java [hunk 1]
```diff
@@ -86,6 +88,8 @@ protected List<WebElement> getElementsByXpath(String locator) {
     public static void scrollToElement() {
```
Comment (DashaYab) [inline src/test/java/pages/BasePage.java:89]: Why method is empty?

File: src/test/java/pages/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public WebElement getMcDelivery() {
+        return driver.findElement(By.xpath("//div[@class='cmp-global-header__primary-nav']//a[@href='/ua/uk-ua/mcdelivery.html']"));
```
Comment (DashaYab) [inline src/test/java/pages/HomePage.java:13]: Move to another line this Xpath, please. It is too long and out of bounds....

File: src/test/java/tests/TestInit.java [hunk 3]
```diff
@@ -25,7 +25,7 @@ public void setup() {
 
     @AfterMethod
     public void closeTest() {
-        driver.quit();
+//        driver.quit();
```
Comment (VolodymyrHrytsiuk) [inline src/test/java/tests/TestInit.java:28]: Why is the line commented out?

File: src/test/java/pages/BasePage.java [hunk 4]
```diff
@@ -4,10 +4,12 @@
 import org.openqa.selenium.*;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
```
Comment (VolodymyrHrytsiuk) [inline src/test/java/pages/BasePage.java:7]: Imports must be logically separated

File: src/test/java/pages/BasePage.java [hunk 5]
```diff
@@ -4,10 +4,12 @@
...
 import java.time.Duration;
 import java.util.List;
 
+import static org.openqa.selenium.support.ui.ExpectedConditions.elementToBeClickable;
+
+
```
Comment (VolodymyrHrytsiuk) [inline src/test/java/pages/BasePage.java:12]: Delete empty line, please

**Pull request merged**